style.css

```diff
@@ -574,6 +575,133 @@ textarea {
   display: none;
 }
 
+.pop-contain {
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  position: fixed;
+  background-color: rgb(0, 0, 0, 30%);
+  display: flex;
+  justify-content: center;
+}
+
+/* styling for popup window */
+
+.mainPopup {
+  background-color: #fff;
+  width: 90%;
+  margin: 0 auto;
+  height: 700px;
+  margin-top: 50px;
+  border-radius: 10px;
+}
```

Comment:
- [ ] I would kindly suggest you stick to the Figma design as much as possible. Currently, your pop-up cards are not well positioned both in the `mobile` and `desktop` versions respectively. Kindly compare what you have in the Figma template with what you have in your design 👍 👍 

### Highlight
  - Kindly give your desktop pop-up modal card some margin-bottom away from the wall
  - Also I noticed the content on the main page is still showing while the pop-up modal card is displayed which is not meant to be like that. kindly give the pop-up card `container` some `CSS property` to that effect
  e.g ```backdrop-filter: blur(8px);```
  - The mobile version of the pop-up modal card is missing some of the content due to wrong `alignment` and positioning. could you please fix that too

 - Below is a video recording where I point out some errors that need your attention 👇 
 
[screen-capture.webm](https://user-images.githubusercontent.com/78634841/209331970-67e7e84a-ce15-4481-86cc-0dbf9f99a52c.webm)